src/lava/magma/compiler/builder/builder.py

```diff
@@ -28,7 +32,8 @@
     AbstractPyPort,
     PyInPort,
     PyOutPort,
-    PyRefPort, PyVarPort,
+    PyRefPort,
+    PyVarPort
 )
 from lava.magma.compiler.channels.interfaces import AbstractCspPort, Channel, \
     ChannelType
```

Comment:
See the next Todo comment. While you are at factoring out the AbstractProcBuilder class you could also more generic attributes like _proc_model, _model_id, _proc_params out to the abstract class.

Reply:
ACK